shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -76,6 +114,7 @@ - (instancetype)initWithBridge:(shell::AccessibilityBridge*)bridge uid:(int32_t)
   if (self) {
     _bridge = bridge;
     _uid = uid;
+    _container = [[SemanticsObjectContainer alloc] initWithSemanticsObject:self bridge:bridge];
```

Comment:
If a good chunk of nodes won't have children, it might be worth leaving this nil until needed.